src/components/Resume/ResumeDetail/ResumeDetail.jsx

```diff
@@ -0,0 +1,116 @@
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import MakeStyles from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import useFetch from "./ResumeApi";
+
+import React, { useState } from "react";
+import "./ResumeDetail.css";
+
+const useStyles = MakeStyles((theme) => ({
+  margin: {
+    "& > *": {
+      margin: theme.spacing(1),
+    },
+  },
+  root: {
+    width: 500,
+  },
+}));
+// resumeId = resume api link
+function Resume({ resumeId }) {
+  const classes = useStyles();
+  const { data, error } = useFetch(resumeId);
+  const [readOnly, setReadOnly] = useState(true);
+  const [showEdit, setShowEdit] = useState("Edit");
+
+  const edit = (event) => {
+    event.preventDefault();
+    setReadOnly(!readOnly);
+    if (readOnly) {
+      setShowEdit("Save");
+      alert("You can edit"); // change alert message
+    } else {
+      setShowEdit("Edit");
+      alert("Saved"); // change alert message
+    }
+  };
+  if (error) return <div>ERROR!!! Resume not found</div>;
+  return data ? (
```

Comment:
Confusing if there's no data it means is loading? 🤔 
maybe you can add a loading state in your hook to know if the data is loading or not